middleware/validationMiddleware.js

```diff
@@ -150,7 +167,7 @@ export const validateMDParams = withValidationErrors([
     .isNumeric()
     .withMessage('invalid medical history number format')
     .custom(async (medical_history_no) => {
-      const patient = await BasicInfo.findOne({ medical_history_no })
+      const patient = await medcases.findOne({ medical_history_no })
```

Comment:
u shouldn't change the basic info validator to medcases
add a new medcase validator instead